File: data_monitor/src/main/java/docker_monitor/DM_app/constants/Group.java
package docker_monitor.DM_app.constants;

public enum Group {
    CPU("Cpu"),
    IO("IO"),
    MEMORY("Memory"),
    NET("Net");

    private final String val;

    Group(String val) {
        this.val = val;
    }
    @Override
    public String toString() {
        return val;
    }
}

File: data_viewer/data_viewer/Model/Notification/Group.cs
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace data_viewer.Model.Notification
{
    [JsonConverter(typeof(StringEnumConverter))]  
    public enum Group
    {
        [Display(Name = "Cpu")]
        CPU,
        [Display(Name = "Io")]
        IO,
        [Display(Name = "Memory")]
        MEMORY,
        [Display(Name = "Net")]
        NET
    }
}